apps/marketplace/src/common/constants.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * NEWM External Links and Support
+ */
+export const NEWM_DISCORD_URL = "https://discord.com/invite/z4sYqxqgvQ";
+export const NEWM_IO_URL = "https://newm.io/";
+export const NEWM_MARKETPLACE_FAQ_URL = "";
+export const NEWM_MARKETPLACE_SUPPORT_URL = "";
+export const NEWM_MARKETPLACE_TERMS_OF_SERVICE_URL = "";
```

Comment:
There is another tasking to fill these URLs in once they are addressed on Product/Legal side.

https://projectnewm.atlassian.net/browse/MRKT-37